src/io/flutter/FlutterUtils.java

```diff
@@ -127,7 +127,7 @@ public static boolean isNewAndroidStudioProjectWizard() {
       ApplicationInfo info = ApplicationInfo.getInstance();
       int major = Integer.parseInt(info.getMajorVersion());
       int minor = Integer.parseInt(info.getMinorVersion());
-      return major >= 4 && minor >= 2;
+      return true; //major >= 4 && minor >= 2;
```

Comment:
Is this a real change or for testing?

Reply:
Both. It's related to your previous question. I need to refactor this since we no longer need to use the old NPW anymore, but it's not high on the list.